### ISSUE TITLE
Managing RDS data: seeds and samplers

### ISSUE THREAD

**Issue by berniehogan** [codaco/Network-Canvas#248]
The Network Canvas App should be able to handle some aspects of RDS case management. There are several issues to consider: 
1. The use of a 'seed' number on the first page. 
- - Should this somehow be verified? If so, should we be generating seed values that are non-sequential at the end of interviews to be used on cards? 
2. The use of a user-configurable alter sampler for snowball sampling 
- - This is also of use for instances such as RADAR where <N alters are denoted for futher questioning.

**Comment by jthrilly** [comment]
@berniehogan - thanks for this! Could you rephrase the above in terms of changes to the app, or new features that we don't currently have? I just need some help to get my head around it.
